tests/test_override_aperturesigns.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import pytest
+from pyspextool.extract.override_aperturesigns import override_aperturesigns
+from pyspextool.setup_utils import pyspextool_setup
+from pyspextool.extract import config as extract
```

Comment:
```suggestion
from pyspextool.extract import config as config_extract
```

Reply:
How about `config_exctract`?  Renaming something to be completely different is ...confusing.